modules/steam/hm.nix

```
@@ -0,0 +1,39 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.stylix.targets.steam;
+in {
+  options.stylix.targets.steam = {
+    enable = config.lib.stylix.mkEnableTarget "Steam" true;
+    adwaitaForSteam.enable = config.lib.stylix.mkEnableTarget "Adwaita for Steam" false;
```

For discoverability, we might want to consider enabling this option by default:

```suggestion
    adwaitaForSteam.enable = config.lib.stylix.mkEnableTarget "Adwaita for Steam" true;
```

---

We should consider that not everyone likes GTK and silently applying a theme always carries the risk that the theme has bugs and these are then reported to Steam. But since users of Stylix are knowingly theming their desktop, this should be fine because they will know to disable it.